collections/ansible_collections/purestorage/flashblade/docs/purefb_remote_cred.rst

```diff
@@ -0,0 +1,141 @@
+
+purefb_remote_cred -- Create, modify and delete FlashBlade object store remote credentials
+==========================================================================================
+
+.. contents::
+   :local:
+   :depth: 1
+
+
+Synopsis
+--------
+
+Create, modify and delete object store remote credentials
+
+You must have a correctly configured remote array or target
+
+This module is **not** idempotent when updating existing remote credentials
+
+
+
+Requirements
+------------
+The below requirements are needed on the host that executes this module.
+
+- python >= 2.7
+- purity_fb >= 1.1
+- netaddr
+- pytz
+
+
+
+Parameters
+----------
+
+  access_key (optional, str, None)
+    Access Key ID of the S3 target
+
+
+  fb_url (optional, str, None)
+    FlashBlade management IP address or Hostname.
+
+
+  target (optional, str, True)
+    Define whether to initialize the S3 bucket
+
+
+  secret (optional, str, None)
+    Secret Access Key for the S3 or Azure target
+
+
+  state (optional, str, present)
+    Define state of remote credential
+
+
+  api_token (optional, str, None)
+    FlashBlade API token for admin privileged user.
+
+
+  name (True, str, None)
+    The name of the credential
+
+
+
+
+
+Notes
+-----
+
+.. note::
+   - This module requires the ``purity_fb`` Python library
+   - You must set ``PUREFB_URL`` and ``PUREFB_API`` environment variables if *fb_url* and *api_token* arguments are not passed to the module directly
+
+
+
+
+Examples
+--------
+
+.. code-block:: yaml+jinja
+
+    
+    - name: Create NFS offload target
+      purefa_offload:
+        name: nfs-offload
+        protocol: nfs
+        address: 10.21.200.4
+        share: "/offload_target"
+        fa_url: 10.10.10.2
+        api_token: e31060a7-21fc-e277-6240-25983c6c4592
+    
+    - name: Create S3 offload target
+      purefa_offload:
+        name: s3-offload
+        protocol: s3
+        access_key: "3794fb12c6204e19195f"
+        bucket: offload-bucket
+        secret: "wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY"
+        placement: aws-standard-class
+        fa_url: 10.10.10.2
+        api_token: e31060a7-21fc-e277-6240-25983c6c4592
+    
+    - name: Create Azure offload target
+      purefa_offload:
+        name: azure-offload
+        protocol: azure
+        secret: "wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY"
+        container: offload-container
+        account: user1
+        fa_url: 10.10.10.2
+        api_token: e31060a7-21fc-e277-6240-25983c6c4592
+    
+    - name: Delete offload target
+      purefa_offload:
+        name: nfs-offload
+        protocol: nfs
+        state: absent
+        fa_url: 10.10.10.2
+        api_token: e31060a7-21fc-e277-6240-25983c6c4592
+    
```

Comment:
These examples are incorrect and should be changed to `purefb_remote_cred` as listed in the `purefb_remote_cred.py`